.github/workflows/fleather.yml

```diff
@@ -1,6 +1,10 @@
 name: build
 
-on: [push, pull_request]
```

Comment:
I find it interesting to have the CI being run on a the branch before creating the pull request

Reply:
The reason for limiting push to master branch was the duplicated run of CI on PR but I think using push as trigger would solve it and enable running CI on every branch without PR too. I'll test it.